test/src/main/java/org/axonframework/test/aggregate/FixtureConfiguration.java

```diff
@@ -108,15 +107,6 @@
      */
     FixtureConfiguration<T> registerRepositoryProvider(RepositoryProvider repositoryProvider);
 
-    /**
```

Comment:
Not entirely sure why this has been made redundant all of the sudden. Do users still have the option to provide their own `DeadlineManager` now?

Reply:
No, now we have `StubDeadlineManager` which allows time travelling :) and is sufficient for tests.

Reply:
Ah that's fair...makes sense. Thanks for explaining!